apps/question-bank/src/test/java/gov/cdc/nbs/questionbank/entity/WaUiMetadataTest.java

```diff
@@ -12,271 +13,709 @@
 import gov.cdc.nbs.questionbank.entity.question.TextQuestionEntity;
 import gov.cdc.nbs.questionbank.entity.question.WaQuestion;
 import gov.cdc.nbs.questionbank.page.command.PageContentCommand;
+import gov.cdc.nbs.questionbank.page.command.PageContentCommand.QuestionUpdate;
+import gov.cdc.nbs.questionbank.page.command.PageContentCommand.UpdateCodedQuestion;
+import gov.cdc.nbs.questionbank.page.command.PageContentCommand.UpdateDateQuestion;
+import gov.cdc.nbs.questionbank.page.command.PageContentCommand.UpdateNumericQuestion;
+import gov.cdc.nbs.questionbank.page.command.PageContentCommand.UpdateTextQuestion;
+import gov.cdc.nbs.questionbank.page.content.PageContentModificationException;
 import gov.cdc.nbs.questionbank.page.exception.AddQuestionException;
 import gov.cdc.nbs.questionbank.question.command.QuestionCommand.Update;
+import gov.cdc.nbs.questionbank.question.request.QuestionRequest.ReportingInfo;
 import gov.cdc.nbs.questionbank.support.QuestionEntityMother;
 
 @ExtendWith(MockitoExtension.class)
```

Comment:
nit:  I don't think this is doing anything.

Reply:
you sir, are correct